.github/workflows/docs-ci.yaml

```diff
@@ -0,0 +1,54 @@
+jame: documentation CI
+
+on:
+  workflow_call:
+    inputs:
+      git_tag:
+        type: string
+        default: latest
```

Comment:
Do we need a default?
`latest` tag does not exists, and I suppose the `make` command will fail

Reply:
can we have something like

```yaml
on:
  push:
    tags:
      - 'v*'
  workflow_dispatch:
    inputs:
      tag:
        description: 'NO tag'
        default: 'v24.1.1'
        required: true
```